lib/gooddata/rest/connection.rb

```diff
@@ -382,6 +383,47 @@ def get(uri, options = {}, &user_block)
         request(:get, uri, nil, options, &user_block)
       end
 
+      def stream(uri, filename)
+        uri = URI.join(server_url, uri.to_s)
+        puts "Streaming URL #{uri.to_s}"
```

Comment:
Lint/StringConversionInInterpolation: Redundant use of `Object#to_s` in interpolation.